native-sql-engine/core/src/main/scala/com/intel/oap/expression/ColumnarUDF.scala

```diff
@@ -18,9 +18,9 @@
 package com.intel.oap.expression
 
 import com.google.common.collect.Lists
+import com.intel.oap.GazellePluginConfig
 import org.apache.arrow.gandiva.expression.{TreeBuilder, TreeNode}
 import org.apache.arrow.vector.types.pojo.ArrowType
-
```

Comment:
nit: This empty line should not be deleted.

Reply:
Just reverted the change. Thanks!